INSTALL.md

```diff
@@ -1,204 +1,67 @@
-# Installation and Configuration for cfgov-refresh
+# Installation and Congiguration for cfgov-refersh with Vagrant
 
-## 1. Back-end setup
+## 0. Prerequisites
```

Comment:
Actually, since these are the only install steps, merge all these prerequisites into the project dependencies list in the README here --> https://github.com/cfpb/cfgov-refresh/blob/flapjack/README.md#dependencies